oneview_redfish_toolkit/api/scmb.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (2017) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import json
+import logging
+import os
+import ssl
+
+from hpOneView.exceptions import HPOneViewException
+import pika
+from pika.credentials import ExternalCredentials
+
+
+from oneview_redfish_toolkit import util
+
+
+ONEVIEW_CA = "certs/oneview_ca.pem"
+SCMB_CERT = "certs/oneview_scmb.pem"
+SCMB_KEY = "certs/oneview_scmb.key"
+
+
+def check_cert_exist():
```

Comment:
is this the same as `return os.path.isfile(ONEVIEW_CA) & os.path.isfile(SCMB_CERT) & os.path.isfile(SCMB_KEY)`? Will we need a different treatment for each case?

Reply:
fixed